tensorflow_datasets/url_checksums/deep_weeds.txt

```diff
@@ -1,2 +1,2 @@
-https://nextcloud.qriscloud.org.au/index.php/s/a3KxPawpqkiorST/download 935276050 d616b33efe097909bdac9623abc5705b59ad11f6796a26a956c1aa5de652f1ba
+https://docs.google.com/uc?export=download&id=1xnK3B6K6KekDI55vwJ0vnc2IGoDga9cj 935276050 d616b33efe097909bdac9623abc5705b59ad11f6796a26a956c1aa5de652f1ba
```

Comment:
Are you sure that the older and newer checksums are exactly the same except the link part?
We need to register the checksums locally whenever a dataset is added or an older dataset's source is changed. Refer to [this](https://www.tensorflow.org/datasets/add_dataset#test_your_dataset) for registering checksums in TFDS.

Reply:
@NikhilBartwal This is actually my first time contributing to tfds. I read the docs but couldnt understand how i can generate checksums.

Reply:
@PrattJena We all started from somewhere, it's great to have you here!

After editing the dataset file, you should delete the `deep weeds checksums.txt` file and use [this](https://www.tensorflow.org/datasets/add_dataset#run_the_generation_code) command from the terminal i your repo without the `module-import` (Since the dataset in defined internally).

For e.g. if the repo was cloned inside 'datasets' folder, this command would do:
`F:\tfds\datasets>python -m tensorflow_datasets.scripts.download_and_prepare --register_checksums --datasets=deep_weeds`

 This would generate the new dataset and register the checksums.

Reply:
@NikhilBartwal Thank you for your help. I successfully downloaded the images on running the given code. Pls look into the following changes